XenAdmin/Wizards/BootModesControl.Designer.cs

```diff
@@ -64,9 +64,7 @@ private void InitializeComponent()
             // radioButtonBIOSBoot
             // 
             resources.ApplyResources(this.radioButtonBIOSBoot, "radioButtonBIOSBoot");
-            this.radioButtonBIOSBoot.Checked = true;
             this.radioButtonBIOSBoot.Name = "radioButtonBIOSBoot";
-            this.radioButtonBIOSBoot.TabStop = true;

```

Comment:
Refine the code
1. Only check the default radio when necessary
2. Remove the unnecessary set on TabStop(radio button will set tabstop to true when checked)